Gemfile

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 source "https://rubygems.org"
 
+gem "rb-readline"
```

Comment:
Why are we adding rb-readline to the gemfile here?

Reply:
@rhass : I was facing error `'require': cannot load such file — readline (LoadError)` while running `knife google server list/create/delete/show` commands. Other `knife google` commands was working fine without `readline` gem.